lighthouse-core/gather/gatherers/inspector-issues.js

```diff
@@ -28,25 +36,28 @@ class InspectorIssues extends Gatherer {
   }
 
   /**
-   * @param {LH.Gatherer.PassContext} passContext
+   * @param {LH.Gatherer.FRTransitionalContext} context
    */
-  async beforePass(passContext) {
-    const driver = passContext.driver;
-    driver.on('Audits.issueAdded', this._onIssueAdded);
-    await driver.sendCommand('Audits.enable');
+  async startSensitiveInstrumentation(context) {
```

Comment:
I'm thinking this one isn't sensitive. Any compelling reason to keep in sensitive?

Reply:
We've had CSP issues logged for things Lighthouse injects into the page https://github.com/GoogleChrome/lighthouse/issues/11862

Reply:
If that's it then `startInstrumentation` should be fine. Both instrumentation and sensitiveInstrumentation end before `getArtifact`. sensitiveInstrumentation is only a concern if the process of *ending* other instrumentation could show up in results.